platform/broadcom/saibcm-modules/debian/changelog

```diff
@@ -1,3 +1,9 @@
+opennsl (4.2.1.3-1) unstable; urgency=medium
```

Comment:
is this 4.2.1.3-1 correct?  should it be 4.3.0.10-1 instead?

Reply:
will wait for more comments before making the change.

Reply:
this should actually be 6.5.21.